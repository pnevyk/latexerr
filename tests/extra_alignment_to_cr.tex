\documentclass{article}

\begin{document}

\begin{table}
  \begin{tabular}{c}
    Foo & Bar \\
  \end{tabular}
\end{table}

\end{document}
