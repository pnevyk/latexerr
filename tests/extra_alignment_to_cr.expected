File: extra_alignment_to_cr.tex

Error on line 7: There are more &'s than should be in a aligned environment (table, etc.) near Foo &.
